packages/camera/camera_android_camerax/README.md

```diff
@@ -1,3 +1,93 @@
-# camera_android_camerax
+# camera\_android\_camerax
 
-An implementation of the camera plugin on Android using CameraX.
+An Android implementation of [`camera`][1] that uses the [CameraX library][2].
+
+*Note*: This package is under development.
+See [missing features and limitations](#missing-features-and-limitations).
+
+## Usage
+
+This package is [non-endorsed][3]; the endorsed Android implementation of `camera`
+is [`camera_android`][4]. To use this implementation of the plugin instead of
+`camera_android`, you will need to specify it in your `pubsepc.yaml` file as a
+dependency in addition to `camera`:
+
+```yaml
+dependencies:
+  # ...along with your other dependencies
+  camera: ^0.10.4
+  camera_android_camerax: ^0.5.0
+```
+
+## How this plugin accesses Android libraries
```

Comment:
I think we wanted to start putting this kind of information in a separate `CONTRIBUTING.md` file.

cc @stuartmorgan FYI